etc/checkstyle-suppressions.xml

```diff
@@ -20,4 +20,6 @@
               files="io[/\\]kroxylicious[/\\]kubernetes[/\\]filter[/\\]api[/\\].*\.java"/>
     <suppress checks=".*"
               files="io[/\\]kroxylicious[/\\]microbenchmarks[/\\]jmh_generated[/\\].*"/>
+    <suppress checks=".*"
```

Comment:
I guess this one would also cover the `kroxylicious[/\\]kubernetes[/\\]` suppressions above, so I'll remove those.